kustomize/commands/internal/util/configmapSecretFlagsAndArgs_test.go

```diff
@@ -1,15 +1,14 @@
 // Copyright 2019 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
non-blocking nit: 2023

Reply:
copyright should only be updated when net-new files are added. Existing files should keep their original copyright date.